server/src/config.ts

```diff
@@ -0,0 +1,8 @@
+import * as dotenv from "dotenv";
+
+dotenv.config();
```

Comment:
Hint: the [preload](https://github.com/motdotla/dotenv#preload) option is what I was talking about.